base/shell/explorer/taskswnd.cpp

```diff
@@ -1447,14 +1452,17 @@ class CTaskSwitchWnd :
         UpdateButtonsSize(FALSE);
 
 #if DUMP_TASKS != 0
-        SetTimer(hwnd, 1, 5000, NULL);
+        SetTimer(1, 5000, NULL);
 #endif
+        SetTimer(TASK_REFRESH_TIMER, REFRESH_INTERVAL, NULL);
         return TRUE;
     }
 
     LRESULT OnDestroy(UINT uMsg, WPARAM wParam, LPARAM lParam, BOOL& bHandled)
     {
         m_IsDestroying = TRUE;
+        KillTimer(1);
```

Comment:
This `KillTimer(1);` call must be inside a `#if DUMP_TASKS != 0 .... #endif` block.